ush/ioda/bufr2ioda/bufr2ioda_subpfl_argo_profiles.py

```diff
@@ -20,15 +20,13 @@
 from pyioda import ioda_obs_space as ioda_ospace
```

Comment:
Minor issue: I don't know anything about that script, but your text editor is poorly setup. You're adding spaces a the end of a few lines below.

Reply:
Fixed.